src/openvic-simulation/country/Country.cpp

```diff
@@ -24,17 +24,33 @@ CountryParty::CountryParty(
 	policies { std::move(new_policies) } {}
 
 Country::Country(
-	std::string_view new_identifier, colour_t new_colour, GraphicalCultureType const& new_graphical_culture,
-	IdentifierRegistry<CountryParty>&& new_parties, unit_names_map_t&& new_unit_names, bool new_dynamic_tag,
-	government_colour_map_t&& new_alternative_colours
-) : HasIdentifierAndColour { new_identifier, new_colour, false }, graphical_culture { new_graphical_culture },
-	parties { std::move(new_parties) }, unit_names { std::move(new_unit_names) }, dynamic_tag { new_dynamic_tag },
-	alternative_colours { std::move(new_alternative_colours) } {}
+	std::string_view new_identifier,
+	colour_t new_colour,
+	GraphicalCultureType const& new_graphical_culture,
+	IdentifierRegistry<CountryParty>&& new_parties, 
+	unit_names_map_t&& new_unit_names, 
+	bool new_dynamic_tag,
+	government_colour_map_t&& new_alternative_colours,
+	colour_t new_primary_unit_colour,
+	colour_t new_secondary_unit_colour,
+	colour_t new_tertiary_unit_colour
+) : HasIdentifierAndColour { 
+	new_identifier, new_colour, false }, 
+	graphical_culture { new_graphical_culture },
+	parties { std::move(new_parties) }, 
+	unit_names { std::move(new_unit_names) }, 
+	dynamic_tag { new_dynamic_tag },
+	alternative_colours { std::move(new_alternative_colours) },
+	primary_unit_colour { new_primary_unit_colour },
+	secondary_unit_colour { new_secondary_unit_colour },
+	tertiary_unit_colour { new_tertiary_unit_colour } 
+	{}
 
 bool CountryManager::add_country(
 	std::string_view identifier, colour_t colour, GraphicalCultureType const* graphical_culture,
 	IdentifierRegistry<CountryParty>&& parties, Country::unit_names_map_t&& unit_names, bool dynamic_tag,
-	Country::government_colour_map_t&& alternative_colours
+	Country::government_colour_map_t&& alternative_colours, 
+	colour_t primary_unit_colour, colour_t secondary__unit_colour, colour_t tertiary__unit_colour
```

Comment:
What's the rationale behind the double underscores?

Reply:
That was an error when i was creating the variables, woops

Reply:
fixed it